### ISSUE TITLE
Extract conversion logic to library and publish to npm

### ISSUE THREAD

**Issue by ioveracker** [thatbudakguy/ddb2alchemy#10]
It would be ideal to have this repo publish an npm module that can be pulled in by Alchemy to do the conversion. Right now, this code has been copied in and slightly modified (#9) to get our first release of the DDB import out there.

I'd be happy to discuss the logistics of this move, or if you just want to YOLO it out there that's cool too!

Related thought: eventually I'd like to publish the `alchemy.d.ts` types to npm under the Alchemy org so they can be pulled in here and kept in sync as we make changes.

**Comment by ioveracker** [comment]
I just realized this morning that we can install this directly from GitHub without having to publish to npm. Since the Alchemy app will be the only project pulling this in, I think that makes a lot of sense to go that route instead of publishing to npm.

Simple example:
```
yarn add -E thatbudakguy/ddb2alchemy
```

It sounds like we can also point to a specific commit or semver tag by adding a `#`:
```
yarn add -E thatbudakguy/ddb2alchemy#0.1.0
```

**Comment by ioveracker** [comment]
Indeed, I just published a `v0.1.0` tag here and installed it like so:

```
$ yarn cw add -E thatbudakguy/ddb2alchemy
yarn run v1.22.19
$ yarn workspace @alchemy/client-web add -E thatbudakguy/ddb2alchemy
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
success Saved 1 new dependency.
info Direct dependencies
info All dependencies
└─ ddb2alchemy@0.1.0
✨  Done in 27.02s.
```

We still need to export the conversion functions, but that should be pretty easy. Going to do that now.
